cowbird/utils.py

```diff
@@ -531,9 +535,9 @@ class B(object, metaclass=SingletonMeta):
         b1 is b2    # True
         a1 is b1    # False
     """
-    _instances = {}
+    _instances: Dict[SingletonMetaType, SingletonMetaType] = {}
 
-    def __call__(cls, *args, **kwargs):
+    def __call__(cls, *args: Any, **kwargs: Any) -> "SingletonMeta":
```

Comment:
This is because it is not yet defined at this point (the class is still being defined). 
I could use the `SingletonMetaType` instead. 
Aliases by quotes are weirdly unsupported when defining a new type, but are permitted in signatures and variable instantiations.